### PR TITLE
create contact css

### PR DESCRIPTION
**Pull request by HebaAhmad1**
relates #24

### CODE REVIEW

File: contact/contact.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    margin-top: 21px;
+    margin-bottom: 21px;
+    font-size: 36px;
+}
+.container .part .para {
+    text-align: center;
+    margin-left: 30px;
+    margin-right: 30px;
+    font-size: 28px;
+    margin-top: 5px;
```
Comment (AbdallahGot) [inline contact/contact.css:63]: you can write one line here instead of three lines too.

File: contact/contact.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.container .part{
+    width: 350px;
+    height: 510px;
+    margin-top: 47px;
+    margin-bottom: 57px;
+    text-align: center;
+    margin-left: 10px;
+    /* margin-right: 20px; */
```
Comment (AbdallahGot) [inline contact/contact.css:48]: delete this comment

File: contact/contact.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    text-align: center;
+}
+
+.container .part{
+    width: 350px;
+    height: 510px;
+    margin-top: 47px;
+    margin-bottom: 57px;
+    text-align: center;
+    margin-left: 10px;
```
Comment (AbdallahGot) [inline contact/contact.css:47]: you can write one line here instead of three lines.
like that `margin: 47px 0 57px 10px`

**Review by AbdallahGot — CHANGES_REQUESTED** [review]
and don't leave many empty lines in your code

**Review by NadaSaleh20 — APPROVED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

**Pull request merged**